### DISCUSSION TITLE
"Backup" description incorrect?

### DISCUSSION THREAD

**Discussion by canton7** [TonyM1958/HA-FoxESS-Modbus#15] [General]
Hi,

I don't think the description of "Back-up" is correct: https://github.com/TonyM1958/HA-FoxESS-Modbus/wiki/Inverter-Work-Modes#backup

If you have a scenario where the batteries need charge, and PV alone is not sufficient to power the house load, then the inverter (or my H1 at least) will power as much as the house load as it can from PV, and not charge the battery. (However, it won't discharge the battery).

This is in contrast to your description, which says that PV is first used to charge the battery, and then power house load.

**Comment by TonyM1958** [comment]
The Fox description isn't very informative. I'll test it again when the sun is shinning and update.
Thanks for checking.

**Comment by TonyM1958** [comment]
Pending me running a test - the inverter will use PV to supply the house load if, for example, the battery charge current is limited and the PV power generation exceeds the battery charge power. Just checking this isn't what you are seeing?

Asking, as if you flip the order so the house load is supplied first and then battery is charged, the description for backup mode starts to look the same as Self Use (except, of course, when PV generation drops, the battery doesn't discharge to supply the house load).

**Reply by canton7** [reply]
From what I'm seeing, PV is used to supply house load, then batteries, then export. House load draws first from PV, then grid (and won't draw from batteries).

**Comment by FozzieUK** [comment]
Interesting - I thought it operated as per Tony's description, however the Solax inverter on which the Hybrid was originally based (same developers) describes the Backup mode as 'maintaining the battery capacity' and using the same working logic as Self-Use mode.

Now just need a bit of sun to test it...

**Reply by FozzieUK** [reply]
Ok, confirmed that when testing in backup mode, PV is used to power the house load and the grid makes up the difference - the battery is getting a slight positive charge (~50W) as if it's trying to compensate for operational losses but nothing more.

I assume the battery will start to get more charge when PV exceeds house load but looking at the forecast it might be a day or two before I have enough solar power to test that.

**Reply by FozzieUK** [reply]
also confirmed the assumption that excess PV will charge the battery - so effectively backup mode is **self-use** but with battery preservation

**Reply by canton7** [reply]
Yep, if PV > Load, then the excess goes into the batteries.

**Comment by canton7** [comment]
I think the following is correct?

Mode | PV Priority | Load Priority | Battery Charge Priority
-- | -- | -- | --
Force charge | Load > Battery > Grid | PV > Grid | PV
Force charge from grid | Load > Battery > Grid | PV > Grid | PV > Grid
Self Use | Load > Battery > Grid | PV > Battery > Grid | PV
Feed-in First | Load > Grid > Battery | PV > Battery > Grid | PV
Back-up | Load > Battery > Grid | PV > Grid | PV

**Reply by canton7** [reply]
So yes, according to that Back-up is Self Use but without the ability to draw the load from the battery.

**Reply by canton7** [reply]
Interestingly, that makes Back-up the same as charge periods (without "charge from grid" selected)

**Reply by FozzieUK** [reply]
yep agree with all of that 👍

**Reply by TonyM1958** [reply]
Thanks Guys - was out getting cold and wet today in a quarry so only just catching up with this.

The table looks good, I'd like to use this. I think I'd need to add Force Charge Work Mode and Force Discharge Work Mode - as these are different to Charge from Grid / Force Charge time periods?

**Reply by canton7** [reply]
Force Charge is different -- it will displace PV, whereas normal charge periods won't. I'm not sure how you'd represent that on the table...

I'm also not sure how you'd represent Force Discharge there. We don't have an "Export Priority" column.

**Reply by TonyM1958** [reply]
Agree - I copied the table and then started editing it and found I was changing too much and it wasn't very clear outside the context of each work mode.

In the end, I ran some testing, got the same results as you (expected) and edited the text in the wiki to cover the changes: https://github.com/TonyM1958/HA-FoxESS-Modbus/wiki/Inverter-Work-Modes#backup
